### PR TITLE
Account key upload

### PR DESCRIPTION
**Pull request by slimjim777**

### CODE REVIEW

File: webapp/js/components/Utils.js [hunk 1]
```diff
@@ -23,3 +23,35 @@ export function T(message) {
     return msg
 }
 
+export function isHTTPError(response) {
+    // Check for an HTTP error
+    if ((response.statusCode >= 300) && (response.statusCode != 400)) {
```
Comment (rmescandon) [inline webapp/js/components/Utils.js]: Not sure that 3xx codes can be considered errors, but 400 is indeed an error. What's the purpose of this method? Maybe the name or the conditions to return true/false should be changed?

**Comment by rmescandon** [comment]
Btw, aspect of web is correct. The ugly view was happening only in my laptop

File: webapp/js/components/Utils.js (hunk 1 above)
Comment (slimjim777) [inline webapp/js/components/Utils.js]: The API returns an error 400 for "expected errors" i.e. ones that have been caught and returned as a JSON response. So, the normal responses will always be 200 or 400. Anything else is something unexpected happening at the server and it is likely that the response won't be JSON. So this is to handle the unexpected errors gracefully.

Comment (rmescandon) [inline webapp/js/components/Utils.js]: so isUnexpectedHTTPError() isUnmanagedHTTPError()... names for the function?

Comment (slimjim777) [inline webapp/js/components/Utils.js]: I've found that the function is not used. It's imported in AccountForm.js, but has been replaced with parseError. I'll merge for now and remove it in the next PR, which is in progress.

**Pull request merged**